src/qcodes/dataset/data_set.py

```diff
@@ -1278,18 +1292,12 @@ def _enqueue_results(
                 res_list = [res_dict]
             self._results += res_list
 
-        # Finally, handle standalone parameters
-
-        standalones = set(interdeps.standalones).intersection(set(result_dict))
-
-        if standalones:
```

Comment:
and standalones are now part of the top level parameters? hence this removed code is covered via the new top_level_params attrbiute?

Reply:
Yes that is the idea. top_level_params are parameters with dependencies, Parameters with inferences that are not them self dependencies and standalones.